### DISCUSSION TITLE
Which EPM1270T144 to buy?

### DISCUSSION THREAD

**Discussion by mczero80** [UzixLS/zx-sizif-512#67] [General]
There seem to be different versions available. 

EPM1270T144I5N and EPM1270T144C5N.

Which one is the correct one?

Thank you very much.

**Comment by UzixLS** [comment]
Both are acceptable. Only difference between them is operating temperature range. "C" means "Commercial" range: 0-85°C, "I" means "Industrial": -40-100°C.
![изображение](https://github.com/user-attachments/assets/3f129963-6fd6-4502-bf56-f649a06d24bc)
